packaging/torchvision/meta.yaml

```diff
@@ -18,6 +20,10 @@ requirements:
 
   run:
     - python
+    - libpng
+    - libjpeg-turbo
+    # Pillow introduces unwanted conflicts with libjpeg-turbo, as it depends on jpeg
+    # The fix depends on https://github.com/conda-forge/conda-forge.github.io/issues/673
```

Comment:
Could you explain when / how these conflicts materialize?
Could that lead to segfaults when the user imports torchvision and PIL?

Reply:
The conflict here is that we're substituting the libjpeg library, which is used primarily by Pillow, however this may not affect us as the ABI between jpegturbo and libjpeg is the same AFAIK. 

Most Linux distributions distinguish between libturbojpeg (the library itself) and libjpeg-turbo (the libjpeg version using turbojpeg), thus, they do not have this problem, as we link against libturbojpeg, rather than against libjpeg-turbo.

In conda-forge, we have this conflict because the recipe for libjpeg-turbo produces both the turbo flavored libjpeg and libturbojpeg. I spoke to @isuruf, one of the maintainers in conda-forge, and he told me that they are working towards a solution in https://github.com/conda-forge/conda-forge.github.io/issues/673, but right now there are no alternative to this conflict. 

I guess we are not having any problem with this conflicting installation, as the tests are passing. However, we should encourage users to install PyTorch and torchvision, so we prevent other errors that could be caused as part of this conflict and we're not aware of

Reply:
Hum, this makes me a bit worried, because AFAIK we don't have tests running on OSX, only Linux and Windows (we do compile on OSX though).
So if there is a problem happening in OSX we wouldn't be able to see it in CI.

Do you think that, if we were to use libjpeg API, everything would be safer?

Reply:
Aren't they running on the `binary_macos_conda_*_pyxx` CircleCI pipelines?

Reply:
Oh ok, yeah, we were not running tests for OSX with wheels, only conda.

But still, do you think if we were to use libjpeg API it would make things safer?

Reply:
I think right now we don't have much trouble on our setup, as we are linking against libturbojpeg and not libjpeg directly. If we were compilling against libjpeg, then we would be in trouble. Right now the only conflict that we have is the one on conda-forge. 

A provisional solution would be compilling libturbojpeg (Without libjpeg) ourselves and publish it into the conda pytorch channel until https://github.com/conda-forge/conda-forge.github.io/issues/673 is fixed. What do you think about this?